pkg/defaults/policies/files/exec-iptables-root.json

```diff
@@ -13,38 +14,78 @@
   ],
   "eventSource": "DEPLOYMENT_EVENT",
   "exclusions": [
+    {
```

Comment:
Shorten to:
{ "name": "Don't alert on ovnkube-node deployment in openshift-ovn-kubernetes Namespace",
      "deployment": {
        "name": "ovnkube-node",
        "scope": {
          "namespace": "openshift-ovn-kubernetes",
        }
      },
    },

Reply:
This should help with the misc-checks CI job failure as well